src/components/modules/toolbar/conversion.ts

```diff
@@ -252,6 +246,14 @@ export default class ConversionToolbar extends Module<ConversionToolbarNodes> {
       return;
     }
 
+    /**
+     * If this conversion fired by the one of multiple Toolbox items,
+     * extend converted data with this item's "data" overrides
+     */
+    if (blockDataOverrides) {
+      newBlockData = Object.assign(newBlockData, blockDataOverrides);
```

Comment:
There might be a problem if override is not on the first level.
Eg.
```
// data
{
  settings: {
    level: 1, 
    anotherProp: 'value',
  }
}

// override
{
  config: {
    level: 2
  }
}
```

Maybe just left a todo to use deep merge in the future

Reply:
We will add a note at the docs describing that data overrides should have the same structure that block data